util/reggen/gen_dv.py

```diff
@@ -39,7 +39,7 @@ def miname(m):
 
 # function get base addr in SV syntax
 def sv_base_addr(b):
-    sv_base_addr = b.base_addr.replace("0x", str(b.width) + "'h")
+    sv_base_addr = str(b.base_addr).replace("0x", str(b.width) + "'h")
```

Comment:
I'm a bit confused about this. Doesn't a block's `base_addr` field have type `int`? But then there's no "0x" to replace. Should it be something more like `"{}'h{:x}".format(b.width, b.base_addr)` ?

Reply:
You're absolutely right. /me is reminded that a quick fix in the middle of a rebasing session isn't necessarily read for a PR :smile: